package.json

```diff
@@ -3,25 +3,30 @@
   "version": "14.2.3",
   "description": "A barebones routing solution for Redux applications.",
   "main": "lib/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
   "scripts": {
     "preversion": "npm run check",
-    "version": "npm run clean && npm run build && git add -A dist",
-    "demo": "webpack-dev-server --port 3000 --config config/webpack/demo/webpack.config.demo.dev.js --colors",
-    "build-demo": "webpack --bail --config config/webpack/demo/webpack.config.demo.js",
-    "start-ssr-demo": "nodemon --watch demo demo/server/index.js",
+    "version": "npm run clean && npm run build",
+    "demo": "BABEL_ENV=commonjs webpack-dev-server --port 3000 --config config/webpack/demo/webpack.config.demo.dev.js --colors",
+    "build-demo": "BABEL_ENV=commonjs webpack --bail --config config/webpack/demo/webpack.config.demo.js",
```

Comment:
I don't think we need `BABEL_ENV=commonjs` for anything pure webpack -- so not for `demo` or `build-demo`

Reply:
The demo actually didn't work unless it was with the `commonjs` env. When I went with the `es` build for demo (which is the default now), it complained about `export` not being a recognized token.

Reply:
Hmmmm... Let me take a look at this and figure out what's going wrong...